product-configuration-link/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,6 @@
+[[extensions]]
+name = "{{ name }}"
+type = "product_configuration_link"
+handle = "{{ handle }}"
+{% if uid %}uid = "{{ uid }}"{% endif %}
+pattern = "/bundles/{product_id}"
```

Comment:
> "/bundles{?product_id,shop}&id={contract_id}"

we will need this to include shop & contract_id as query parameters, similar to https://github.com/Shopify/extensions-templates/pull/162/files#diff-ab9067fa02fd15d1e13957d76d8f11a774bd0132a0df810ae7d1f494021b4ede